ci/publish-artifactory.sh

```diff
@@ -23,7 +23,7 @@ push() {
          -H "X-Checksum-SHA1:${sha1}" \
          -X PUT \
          -T ${local_path} \
-         https://digitalasset.jfrog.io/artifactory/${repository}/$(release_tag)/${file}
+         https://digitalasset.jfrog.io/artifactory/${repository}/$RELEASE_TAG/${file}
```

Comment:
Would it make sense for this to be quoted? I only bring this up because it's the only thing I learned from Shellcheck. 😄

Reply:
I would say "no" because we've built all of those to not have spaces in them, and I'd rather have the script fail on some syntax error than push files with spaces in their name should we ever screw that up. Overall I must say I've been unimpressed with most shellcheck suggestions when I've looked at it.

But I know @SamirTalwar-DA for example disagrees and prefers quoting things in general, so YMMV. 🙂

Reply:
I think I agree with @garyverhaegen-da here. We assume that there are no spaces here in all kinds of places. Adding quotes in a few select places isn’t super helpful.